openwisp_controller/test_signal.py

```diff
@@ -0,0 +1,11 @@
+from contextlib import contextmanager
```

Comment:
please move it here for consistency: https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/tests/utils.py

Reply:
Done @nemesisdesign :smile:. After this working on [openwisp-utils#87](https://github.com/openwisp/openwisp-utils/issues/87)